docs/psa_switch-dev.md

```diff
@@ -0,0 +1,82 @@
+# Implementing Externs in PSA Switch
+Externs in PSA Switch are different than those in Simple Switch. 
```

Comment:
I think this would be more accurate as "The implementation of P4 externs in PSA Switch is different..."